.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,13 +1,27 @@
-<!--  Delete any sections below that are not relevant to this PR -->
+<!-- Insert GitHub issue number below -->
+Fixes #
 
 ## Description
-<!--- What are the changes? -->
+<!-- Briefly describe the proposed changes -->
 
-## Context
-<!--- Why did you make these changes, and why in this particular way? -->
+## Notes
+<!-- Be sure to mention anything unusual, out-of-scope or new technical debt, etc -->
 
-## References
-<!-- Include links to JIRA, Github, etc. if appropriate -->
+## Checklist
+<!-- Acknowledge the general checklist below -->
+- [ ] I verify the build is in a non-broken state
+- [ ] I verify all changes are within scope of the linked issue
+
+<!-- Acknowledge which type of PR this is -->
+- [ ] This PR contains CSS changes
+- [ ] This PR does not contain CSS changes
```

Comment:
Is this needed? You can probably tell by the changes. Most of the time it will be css changes.

Reply:
I was experimenting with some branching options in the checklist. it also helps reviewer quickly tell the scope (e.g. simple doc fixes vs whole new feature). Labels could also work for that, although people often forget them. Different PR templates would probably be best, but was seeing how far I could go without them...

Reply:
We could maybe just run with this experiment and see how it goes..? It's trivial to update it at any time or change to separate templates.

I think I've also sneakily gone this route as it helps prevent PRs from scope creep. If people try and sneak some quick css changes into something that should only be a doc change, then they also now have to be prepared to fill out the CSS checklist part.

Reply:
Maybe we can have this be different from the checklist then. From what I know the checklist is a reminder of things to do and not what the PR is about. 
Maybe we can have this as comment and say:
## Pull Requets Type
<!-- valid types are, CSS, script, build -->

Reply:
Yes, I mean we are free to define our own rules and whatever makes sense for us. It doesn't just have to be a reminder of things to do, we can use it to provide signals like this. The definition of type has merit, I just worry it could get too long and be too many combinations, e.g. css, docs and storybook.

Reply:
But the same thing with checklist, are we going to list out all the types of PRs in a checklist? 
Maybe we shoud use more labels. I think that makes the most sense. I haven't been becasue we usually have labels on the issues (and it seems redundant to have the same labels on the PR/Issues). 
But maybe we should have: `PR Type` labels. I think that would be helpful to have (and easier to search for)

Reply:
I was hoping not to, and instead wanted to see if just a simple "contains CSS" or "doesn't contain CSS" would work, because that is the main factor that drives the checklist. For PRs that don't contain CSS changes, it's a pretty quick and easy checklist.